src/Sinch/Verification/SinchVerification.cs

```diff
@@ -20,7 +21,56 @@ public interface ISinchVerification
         /// <param name="request"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        Task<IVerificationStartResponse> Start(VerificationStartRequest request,
+        Task<IStartVerificationResponse> Start(StartVerificationRequest request,
```

Comment:
Do we need to keep this method if all the 4 possibilities are covered below?

Reply:
yeah, makes sense - now this method is an overhead